tests/cypress/integration/regTest_for_export_task/innerExportTask.js

```diff
@@ -0,0 +1,138 @@
+// Start writing your Cypress tests below!
+// If you're unfamiliar with how Cypress works,
+// check out the link below and learn how to write your first test:
+// https://on.cypress.io/writing-first-test
```

Comment:
We need to put a lincese header here instead of default Cypress text:

// Copyright (C) 2022 CVAT.ai Corporation
//
// SPDX-License-Identifier: MIT